src/sys/windows/selector.rs

```diff
@@ -591,7 +575,8 @@ impl SelectorInner {
     }
 
     unsafe fn update_sockets_events_if_polling(&self) -> io::Result<()> {
```

Comment:
Since active_poll_count has been removed, this function and all its call sites can be removed too.

Reply:
Done.